README.md

```diff
@@ -50,6 +50,7 @@ or adapt the bindings to your needs:
 | `direction`        | `up`, `down`, `left` or `right` | `up`   | The direction to open the action buttons               |
 | `animationMode`    | `fling` or `scale` | `fling`             | The animation to apply when opening the action buttons |
 | `fixed`            | `boolean`          | `false`             | Indicates if this FAB Speed Dial is fixed (user cannot change the open state on click) |
+| `closeOnOutsideClick` | `boolean`       | `false`             | Indicates if this FAB Speed Dial will automatically close when user clicks somewhere in document (does not apply to fixed mode) |
```

Comment:
I think this feature should be the only behavior possible and hence no configuration should be needed at all. This was the case in AngularJS AFAIK.

Is there a specific reason why you made it configurable ?

```suggestion
```

Reply:
I will try to create it as the only behavior possible.